app/views/idv/finance/new.html.slim

```diff
@@ -0,0 +1,15 @@
+- title t('idv.titles.welcome')
+
+.mt1.mb3.h6.caps.red = t('idv.titles.welcome')
+h2.heading = t('idv.titles.session.finance')
+p.italic Please provide only one of the following:
+= simple_form_for(idv_finance_form, url: idv_finance_path,
+    html: { autocomplete: 'off', method: 'put', role: 'form' }) do |f|
+  = f.error_notification
+  = f.collection_radio_buttons :finance_type, Idv::FinanceForm.finance_type_choices,
+    :first, :last, required: true, item_wrapper_tag: :div
```

Comment:
Should we add make the first button selected by default? You can do that by adding `checked: Idv::FinanceForm.finance_type_choices.first`.